__tests__/applyStyle.test.js

```diff
@@ -0,0 +1,233 @@
+/*eslint no-console: "off"*/
+import 'isomorphic-fetch';
+import nock from 'nock';
+nock.disableNetConnect();
```

Comment:
It can.  It's just a style preference thing.  In general, I put global modifications to a package right after import so it's obvious something global about the package has been modified.  I'll move it.